### ISSUE TITLE
File vanishes in portable Zim

### ISSUE THREAD

**Issue by micharlie** [zim-desktop-wiki/zim-desktop-wiki#1311]
I've been using (zim-desktop-wiki-portable-0_69_1) as a study notebook. However, when I clicked on it to open it today I got the message: No such file: F:\zim\Zim Desktop Wiki Portable\App\Config\zim\notebooks.list
Does anyone know how I fix this issue so I can access my student work?
If it cannot be fixed, can you tell me how I can transfer my files to a newer version of Zim portable, without having to write them all out again?
I've checked the FAQ, but this isn't covered there and I was directed here.
Thanks for any help.
Kind regards
MiCharlie

**Comment by jaap-karssenberg** [comment]
1) Check the folder where the notes are stored - if they are all there OK -
if not something went really wrong and you should check your backups
2) In the dialog, just add a "new" notebook and point it towards the folder
with the notes, it should open the existing notebook just fine

-- Jaap

**Comment by jaap-karssenberg** [comment]
Please update the ticket with the outcome - can this issue be closed ?

**Comment by micharlie** [comment]
My apologies for the late response, I ended up in hospital with no internet.

The notes were fine. I'm sorry, I didn't receive your answer in time to try it, so I'm not sure if it would have worked.

When I checked the other files, they all worked. So, I changed the permissions on the notepad pages to 'everyone' to match them. That didn't re-link them, but it did allow me to copy and paste the contents to a new notepad page.....which then re-linked the notebooks. 

If it happens again, I will try your solution as it seems much easier than the route I took.

Thank you for your help.
